libsweep/sweep.c

```diff
@@ -19,10 +19,23 @@ typedef struct sweep_device {
 
 #define SWEEP_MAX_SAMPLES 4096
 
+/*
+ * Struct: sweep_scan
+ * ------------------
+ * Contains the data for sensor readings in a scan
+ * 
+ *  angle: array of azimuth values (1 for each reading)
+ *        (in millidegrees, ie: 241.375deg stored as 241375millideg)
+ *  distance: array of range values (1 for each reading)
+ *        (in centimeters)
+ *  signal_strength: array of signal strength values (1 for each reading)
+ *        (integer values between 0:255, higher is better)
+ *  count: number of sensor readings in the scan 
+ */
```

Comment:
Hm care to remove the comment here - the public API docs should be in the README.md. This is the impl. file.